dags/landsat_scenes_sync/landsat_scenes_gap_report.py

```diff
@@ -196,6 +205,59 @@ def request_usgs_api(url: str):
                 yield future.result()
 
 
+def check_for_stac_files(dest_paths):
+    """
+    If there is a missing stac, either ST or SR, and the stac is present in USGS, consider a not transfer scene
+    """
+    def find_stac_files(prefix):
+
+        s3 = S3(conn_id=CONN_LANDSAT_SYNC)
+
+        scene = prefix.split('/')[-2]
+
+        # Checking SR File in Africa
+        sr_file = s3.list_objects(
+            bucket_name=LANDSAT_SYNC_BUCKET_NAME,
+            region=REGION,
+            prefix=f"{prefix}{scene}_SR_stac.json"
+        )
```

Comment:
Don't list all the objects there, that's super slow.

Just get the actual key and check if it exists (doing a get/head request whatever).

Reply:
the `prefix=f"{prefix}{scene}_SR_stac.json"` will search for a specific key not all files. In this case, I'm looking for a specific SR stac.